src/config/index.ts

```diff
@@ -23,6 +23,7 @@ export type ContactType = {
   replacement_property: ContactProperty;
   place_properties: ContactProperty[];
   contact_properties: ContactProperty[];
+  user_roles_property?: ContactProperty;
```

Comment:
No objection to using the `ContactProperty` for this; but it also doesn't seem like the _best_ fit. Here are a few things I'm finding awkward with the interface.

1. `required` attribute is redundant because we have `user_roles_property` can be undefined
2. `property_name` is redundant because this can only control the user's role
3. `type` is not needed because this isn't validated
4. It is a bit confusing having both `user_role` and `user_roles_property`.  Could we somehow get to one?

This sort of awkwardness exists elsewhere in the config like `replacement_property`. Not sure how much work it would be to make a custom type for this; but just mentioning it.

Reply:
Thanks!